app/controllers/sequences_controller.rb

```diff
@@ -73,7 +73,7 @@ def observation_index # :norobots:
     show_selected_sequences(query, always_index: true)
   end
 
-  def show
+  def show # :norobots:
```

Comment:
I think you can actually get rid of this magic comment per this email:
>Jason Hollinger <pellaea@gmail.com> | 23 de abril de 2019 a las 05:17
_Para:_ Joseph Cohen <jdcohenesq@gmail.com>
CC: Nathan Wilson <nathan@collectivesource.com>
Get rid of them all, I think.  I feel like I was following some pre-existing convention... maybe it was :nodoc:.  But I don't think any, even the one(s) I didn't make up, are important.  :nodoc: is effectively unused since we don't use the autodocumentation stuff.  :norobots: and :prefetch: don't appear to be used either.  Oh look, I actually even wrote something (script/old/refresh_controller.rb) that looks at these directives, but never apparently used it.-Jason